### ISSUE TITLE
[🐛 Bug]: java.lang.NoClassDefFoundError: java/net/http/WebSocket$Listener

### ISSUE THREAD

**Issue by youngformax** [SeleniumHQ/selenium#13111]
### What happened?

When I use selenuim-java in Compose multiplatform, I use `gradle run` to open chromedriver without any problems, but when I use `gradle runDistributable` for distribution testing, this error occurs. How to solve this problem?
```kotlin
@Composable
@Preview
fun App() {
    var text by remember { mutableStateOf("Hello, World!") }

    MaterialTheme {
        Button(onClick = {
            text = "Hello, Desktop!"
            val chromeDriver = ChromeDriver()
            chromeDriver.get("https://www.baidu.com")
        }) {
            Text(text)
        }
    }
}

fun main() = application {
    Window(onCloseRequest = ::exitApplication) {
        App()
    }
}
```

### How can we reproduce the issue?

```shell
Create a compose multiplatform project in IntelliJ IDEA, add selenuim-java dependency, and then run`gradle runDistributable`
```

### Relevant log output

```shell
> Task :runDistributable
Exception in thread "AWT-EventQueue-0" java.lang.NoClassDefFoundError: java/net/http/WebSocket$Listener
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient$Factory.createClient(JdkHttpClient.java:483)
	at org.openqa.selenium.remote.HttpCommandExecutor.<init>(HttpCommandExecutor.java:115)
	at org.openqa.selenium.remote.HttpCommandExecutor.<init>(HttpCommandExecutor.java:92)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.<init>(DriverCommandExecutor.java:89)
	at org.openqa.selenium.chromium.ChromiumDriverCommandExecutor.<init>(ChromiumDriverCommandExecutor.java:44)
	at org.openqa.selenium.chrome.ChromeDriver$ChromeDriverCommandExecutor.<init>(ChromeDriver.java:116)
	at org.openqa.selenium.chrome.ChromeDriver.generateExecutor(ChromeDriver.java:106)
	at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:89)
	at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:84)
	at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:52)
	at MainKt$App$1$1$1.invoke(Main.kt:22)
	at MainKt$App$1$1$1.invoke(Main.kt:20)
	at androidx.compose.foundation.ClickablePointerInputNode$pointerInput$3.invoke-k-4lQ0M(Clickable.kt:898)
	at androidx.compose.foundation.ClickablePointerInputNode$pointerInput$3.invoke(Clickable.kt:891)
	at androidx.compose.foundation.gestures.TapGestureDetectorKt$detectTapAndPress$2$1.invokeSuspend(TapGestureDetector.kt:258)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTaskKt.resume(DispatchedTask.kt:178)
	at kotlinx.coroutines.DispatchedTaskKt.dispatch(DispatchedTask.kt:166)
	at kotlinx.coroutines.CancellableContinuationImpl.dispatchResume(CancellableContinuationImpl.kt:397)
	at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl(CancellableContinuationImpl.kt:431)
	at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$default(CancellableContinuationImpl.kt:420)
	at kotlinx.coroutines.CancellableContinuationImpl.resumeWith(CancellableContinuationImpl.kt:328)
	at androidx.compose.ui.input.pointer.SuspendingPointerInputModifierNodeImpl$PointerEventHandlerCoroutine.offerPointerEvent(SuspendingPointerInputFilter.kt:672)
	at androidx.compose.ui.input.pointer.SuspendingPointerInputModifierNodeImpl.dispatchPointerEvent(SuspendingPointerInputFilter.kt:549)
	at androidx.compose.ui.input.pointer.SuspendingPointerInputModifierNodeImpl.onPointerEvent-H0pRuoY(SuspendingPointerInputFilter.kt:571)
	at androidx.compose.foundation.AbstractClickablePointerInputNode.onPointerEvent-H0pRuoY(Clickable.kt:857)
	at androidx.compose.foundation.AbstractClickableNode.onPointerEvent-H0pRuoY(Clickable.kt:705)
	at androidx.compose.ui.input.pointer.Node.dispatchMainEventPass(HitPathTracker.kt:292)
	at androidx.compose.ui.input.pointer.Node.dispatchMainEventPass(HitPathTracker.kt:278)
	at androidx.compose.ui.input.pointer.NodeParent.dispatchMainEventPass(HitPathTracker.kt:156)
	at androidx.compose.ui.input.pointer.HitPathTracker.dispatchChanges(HitPathTracker.kt:93)
	at androidx.compose.ui.input.pointer.PointerInputEventProcessor.process-BIzXfog(PointerInputEventProcessor.kt:97)
	at androidx.compose.ui.platform.SkiaBasedOwner.processPointerInput-gBdvCQM$ui(SkiaBasedOwner.skiko.kt:362)
	at androidx.compose.ui.platform.SkiaBasedOwner.processPointerInput-gBdvCQM$ui$default(SkiaBasedOwner.skiko.kt:355)
	at androidx.compose.ui.ComposeScene.processRelease(ComposeScene.skiko.kt:652)
	at androidx.compose.ui.ComposeScene.processPointerInput(ComposeScene.skiko.kt:611)
	at androidx.compose.ui.ComposeScene.access$processPointerInput(ComposeScene.skiko.kt:68)
	at androidx.compose.ui.ComposeScene$syntheticEventSender$1.invoke(ComposeScene.skiko.kt:245)
	at androidx.compose.ui.ComposeScene$syntheticEventSender$1.invoke(ComposeScene.skiko.kt:245)
	at androidx.compose.ui.SyntheticEventSender.sendInternal(SyntheticEventSender.kt:157)
	at androidx.compose.ui.SyntheticEventSender.send(SyntheticEventSender.kt:64)
	at androidx.compose.ui.ComposeScene.sendPointerEvent-WlEVilQ(ComposeScene.skiko.kt:581)
	at androidx.compose.ui.ComposeScene.sendPointerEvent-BGSDPeU(ComposeScene.skiko.kt:523)
	at androidx.compose.ui.ComposeScene.sendPointerEvent-BGSDPeU$default(ComposeScene.skiko.kt:506)
	at androidx.compose.ui.awt.ComposeBridge_desktopKt.onMouseEvent(ComposeBridge.desktop.kt:405)
	at androidx.compose.ui.awt.ComposeBridge_desktopKt.access$onMouseEvent(ComposeBridge.desktop.kt:1)
	at androidx.compose.ui.awt.ComposeBridge$onMouseEvent$1.invoke(ComposeBridge.desktop.kt:245)
	at androidx.compose.ui.awt.ComposeBridge$onMouseEvent$1.invoke(ComposeBridge.desktop.kt:238)
	at androidx.compose.ui.awt.ComposeBridge.catchExceptions(ComposeBridge.desktop.kt:126)
	at androidx.compose.ui.awt.ComposeBridge.onMouseEvent(ComposeBridge.desktop.kt:238)
	at androidx.compose.ui.awt.ComposeBridge.access$onMouseEvent(ComposeBridge.desktop.kt:59)
	at androidx.compose.ui.awt.ComposeBridge$attachComposeToComponent$3.mouseReleased(ComposeBridge.desktop.kt:219)
	at java.desktop/java.awt.Component.processMouseEvent(Unknown Source)
	at java.desktop/java.awt.Component.processEvent(Unknown Source)
	at java.desktop/java.awt.Component.dispatchEventImpl(Unknown Source)
	at java.desktop/java.awt.Component.dispatchEvent(Unknown Source)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.desktop/java.awt.EventQueue$4.run(Unknown Source)
	at java.desktop/java.awt.EventQueue$4.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.desktop/java.awt.EventQueue$5.run(Unknown Source)
	at java.desktop/java.awt.EventQueue$5.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.desktop/java.awt.EventQueue.dispatchEvent(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.run(Unknown Source)
	Suppressed: kotlinx.coroutines.DiagnosticCoroutineContextException: [androidx.compose.ui.awt.ComposeBridge$coroutineExceptionHandler$1@12070144, androidx.compose.runtime.BroadcastFrameClock@5bcfae64, StandaloneCoroutine{Cancelling}@6cf1614d, FlushCoroutineDispatcher@205127a]
Caused by: java.lang.ClassNotFoundException: java.net.http.WebSocket$Listener
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(Unknown Source)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	... 73 more
Caused by: java.lang.ClassNotFoundException: java.net.http.WebSocket$Listener
```

### Operating System

Windows 10

### Selenium version

java 4.15.0

### What are the browser(s) and version(s) where you see this issue?

Chrome 119

### What are the browser driver(s) and version(s) where you see this issue?

Chrome 119

**Comment by titusfortner** [comment]
This looks like a dependency conflict. Check out https://www.javaadvent.com/2020/12/how-to-debug-dependency-conflicts-in-maven-and-gradle.html

**Comment by youngformax** [comment]
> This looks like a dependency conflict. Check out https://www.javaadvent.com/2020/12/how-to-debug-dependency-conflicts-in-maven-and-gradle.html

Thanks.I'll try it.

**Comment by grclague** [comment]
@young-kingdom If there was a dependency conflict, did you manage to resolve it? I'm struggling with the same issue.

**Comment by youngformax** [comment]
> @young-kingdom If there was a dependency conflict, did you manage to resolve it? I'm struggling with the same issue.

@grclague Actually in the end I didn't solve the problem, I tried other packaging frameworks.It seems to be a packaging issue with the gradle task `gradle runDistributable` and related release tasks in compose multiplatform. I abandoned the packaging task of compose multiplatform and changed it to javapackager. Then I can use selenuim-java normally.
